base/system/winlogon/sas.c

```diff
@@ -1046,7 +1047,12 @@ HandleShutdown(
     BOOLEAN Old;
 
     // SwitchDesktop(Session->WinlogonDesktop);
-    DisplayStatusMessage(Session, Session->WinlogonDesktop, IDS_REACTOSISSHUTTINGDOWN);
+    
+    /* If the system is rebooting, show IDS_REACTOSISRESTARTING string instead */
```

Comment:
```suggestion
    /* If the system is rebooting, show the according string */
```

Reply:
_According_ feels out of place here (according to what? to whom?). Maybe use _appropriate_ instead?

Reply:
Thanks, corrected